github/filecoin-project.yml

```diff
@@ -3946,59 +3848,32 @@ repositories:
         - proofs-crate-owners
       maintain:
         - proofs
-      pull:
-        - dev-misc
     visibility: public
   rust-fil-nse-gpu:
-    advanced_security: false
-    allow_update_branch: false
-    archived: false
-    collaborators:
-      admin:
-        - porcuquine
-    has_discussions: false
-    merge_commit_message: PR_TITLE
-    merge_commit_title: MERGE_MESSAGE
-    secret_scanning_push_protection: true
-    secret_scanning: true
-    squash_merge_commit_message: COMMIT_MESSAGES
-    squash_merge_commit_title: COMMIT_OR_PR_TITLE
-    teams:
-      admin:
-        - proofs-crate-owners
-    visibility: public
+    archived: true
   rust-fil-proofs-ffi:
     archived: true
-  # NOTE: We don't add create-owners team here because this repo is never published
   rust-fil-proofs-private:
     advanced_security: false
     allow_update_branch: false
     archived: false
-    collaborators:
-      admin:
-        - cryptonemo
     has_discussions: false
     merge_commit_message: PR_TITLE
     merge_commit_title: MERGE_MESSAGE
     secret_scanning_push_protection: false
     secret_scanning: false
     squash_merge_commit_message: COMMIT_MESSAGES
     squash_merge_commit_title: COMMIT_OR_PR_TITLE
+    teams:
+      admin:
+        # NOTE: This repo is never published, but we still use this group for admin.
+
```

Comment:
seeing this blank line show up multiple places in here I'm guessing this is required to make it yaml happy?

Reply:
I don't think so. The automated fix just trimmed this line because it had some empty spaces. I think we should be able to remove it.

```suggestion
```